### PR TITLE
Fix layout of rv_histogram docs

### PR DESCRIPTION
**Pull request by thomaskeck**
Fix for #7022.
Not tested, I have to check online if the layout of the docs is now fine.

### CODE REVIEW

**Comment by thomaskeck** [comment]
I finally convinced scipy to build its docs and could verify that the documentation of rv_histogram looks fine now.
Hence, this can be merged to fix issue #7022

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -5270,36 +5270,46 @@ class rv_histogram(rv_continuous):
     Generates a distribution given by a histogram.
     This is useful to generate a template distribution from a binned datasample.
 
-    %(before_notes)s
+    As a subclass of the `rv_continuous` class, `rv_histogram` inherits from it a collection of generic methods (see `rv_continuous` for the full list),
```
Comment (ev-br) [inline scipy/stats/_continuous_distns.py]: Please wrap the line <79 chars. Rationale: `rv_histgram?` in ipython terminal.

**Comment by ev-br** [comment]
Looks OK indeed: https://2039-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.stats.rv_histogram.html#scipy.stats.rv_histogram

Merging, thanks @thomaskeck

**Comment by josef-pkt** [comment]
Looking at the nice generate docs, I saw

the sentence "The cdf is a linear interpolation of the pdf." in the notes section is not really correct (literally).

maybe something like
"The cdf and ppf are computed by linear interpolation of the points at the bin boundaries."

**Pull request merged**